selfdrive/car/gm/gmcan.py

```diff
@@ -63,7 +63,7 @@ def create_friction_brake_command(packer, bus, apply_brake, idx, near_stop, at_f
   return packer.make_can_msg("EBCMFrictionBrakeCmd", bus, values)
 
 def create_acc_dashboard_command(packer, bus, acc_engaged, target_speed_kph, lead_car_in_sight, fcw):
-  target_speed = min(target_speed_kph, 255)
+  target_speed = min(target_speed_kph, 255) * 16
```

Comment:
Moving this scaling into the DBC means we don't need to scale in openpilot any more. This would scale the factor: `target_speed * 16 / .0625` which is a 256x scaling factor